src/app/modules/dashboard/modules/day-plan/components/day-plan-stats/day-plan-stats.component.scss

```diff
@@ -0,0 +1,87 @@
+@import "colors.scss";
+@import 'layout.scss';
+
+.day-plan-summary-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  border-top-right-radius: 3rem;
+  border-top-left-radius: 3rem;
+  box-shadow: transparentize(#ca2043, 0.5) 0px -2px 20px 0px;
+  padding: 0.25rem 0.5rem;
+  background-color: #f4d2b5;
+  position: sticky;
+  width: 100%;
+  bottom: 0;
+}
+.day-plan-summary-wrapper-hidden {
+  transform: translateY(67%);
+}
+.summary-element {
+  display: flex;
+  flex-direction: column;
+  margin: 0.5rem;
+  width: 80%;
+}
+
+.summary-element-header-wrapper {
+  display: flex;
+
+  :last-child {
+    margin-left: auto;
+  }
+
+  & span {
+    white-space: pre;
+  }
+}
+
+.summary-arrow {
+  background-image: url(~src/assets/images/icons/arrow.png);
+  background-color: transparent;
+  background-repeat: no-repeat;
+  border: none;
+  cursor: pointer;
+  height: 25px;
```

Comment:
wszystkie pixele zamieńmy na rem

Reply:
ale to sa akurat wymiary obrazka. to tak sie nie robi?